Sketch.java

```diff
@@ -1,36 +1,106 @@
 import processing.core.PApplet;
 
 public class Sketch extends PApplet {
-	
-	
+    
+    
   /**
    * Called once at the beginning of execution, put your size all in this method
    */
   public void settings() {
-	// put your size call here
-    size(400, 400);
+    // put your size call here
+    size(600, 600);
   }
 
   /** 
    * Called once at the beginning of execution.  Add initial set up
    * values here i.e background, stroke, fill etc.
    */
   public void setup() {
-    background(210, 255, 173);
+    background(51,204,255);
   }
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
-	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
+      
+    //sun 
+    fill(255, 255, 0);
+    ellipse((float) (width/8), (float) (height/8),(float)
+        (width/5), (float)(height/5));
+
+    //cloud
+    fill(255, 255, 255);
+  ellipse((float) (width/2.1), (float) (height/13.33),(float)
+        (width/3.08), (float)(height/10));    
+    
+    //cloud
+    fill(255, 255, 255);
+   ellipse((float) (width/1.38), (float) (height/13.33),(float)
+        (width/3.08), (float)(height/10));    
+    
+    //house
+    fill(255, 255, 153);
+  rect((float) (width/4.0), (float) (height/2.58),
+        width/2, height/2);    
```

Comment:
fix your indentation.  code should be aligned and indented within the draw method